decidim-core/app/helpers/concerns/decidim/flash_helper_extensions.rb

```diff
@@ -20,13 +20,21 @@ module FlashHelperExtensions
       # Returns a HTML string.
       def alert_box(value, alert_class, closable)
         options = {
-          class: "flash callout #{alert_class}",
+          class: "flash #{alert_class}",
+          data: { "alert-box": "" },
           role: "alert",
           aria: { atomic: "true" }
         }
-        options[:data] = { closable: "" } if closable
+        options[:data] = options[:data].merge({ closable: "" }) if closable
```

Comment:
Braces are not necessary 

```suggestion
        options[:data] = options[:data].merge(closable: "") if closable
```